gdpr/impl_test.go

```diff
@@ -453,43 +479,57 @@ func TestAllowActivitiesGeoAndID(t *testing.T) {
 		{
 			description:           "Unknown vendor test, insufficient purposes claimed, basic enforcement",
 			bidder:                openrtb_ext.BidderAudienceNetwork,
+			bidderCoreName:        openrtb_ext.BidderAudienceNetwork,
 			consent:               "COzTVhaOzTVhaGvAAAENAiCIAP_AAH_AAAAAAEEUACCKAAA",
 			allowBid:              true,
 			passGeo:               true,
 			passID:                true,
 			weakVendorEnforcement: true,
 		},
 		{
-			description: "Pubmatic vendor test, flex purposes claimed",
-			bidder:      openrtb_ext.BidderPubmatic,
-			consent:     "COzTVhaOzTVhaGvAAAENAiCIAP_AAH_AAAAAAEEUACCKAAA",
-			allowBid:    true,
-			passGeo:     true,
-			passID:      true,
+			description:    "Pubmatic vendor test, flex purposes claimed",
+			bidder:         openrtb_ext.BidderPubmatic,
+			bidderCoreName: openrtb_ext.BidderPubmatic,
+			consent:        "COzTVhaOzTVhaGvAAAENAiCIAP_AAH_AAAAAAEEUACCKAAA",
+			allowBid:       true,
+			passGeo:        true,
+			passID:         true,
 		},
 		{
-			description: "Rubicon vendor test, Specific purposes/LIs claimed, no geo claimed",
-			bidder:      openrtb_ext.BidderRubicon,
-			consent:     "COzTVhaOzTVhaGvAAAENAiCIAP_AAH_AAAAAAEEUACCKAAA",
-			allowBid:    true,
-			passGeo:     false,
-			passID:      true,
+			description:    "Pubmatic Alias vendor test, flex purposes claimed",
+			bidder:         "pumatic1",
```

Comment:
There's a typo here so the alias won't be found in the GVL ID map. I suggest fixing the typo and then adding another test case where the alias is found in the GVL ID map but results in a different outcome than if the core bidder GVL ID was used. Something similar to my suggestion above for `TestAllowActivities` on line 248.

Reply:
Done